documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-management-tasks.adoc
+
+[id='proc-setting-broker-limits-{context}']
+
+= Setting limits on brokers using the Kafka Static Quota plugin
+
+[role="_abstract"]
+Use the _Kafka Static Quota_ plugin to set throughput and storage limits on brokers in your Kafka cluster.
+You can set a byte-rate threshold and storage quotas for clients interacting with your brokers.
+You enable the plugin and set limits by configuring the `Kafka` resource.
+
+You can set byte-rate thresholds for producer and consumer bandwidth.
+The total limit is distributed across all clients accessing the broker.
+For example, you can set a byte-rate threshold of 40 MBps for producers.
+If you have two producers running, they are each limited to a throughput of 20 MBps.
+
+Storage quotas throttle Kafka disk storage limits between a soft limit and hard limit.
+Producers are slowed gradually between the soft and hard limit.
+The limits prevent disks filling up too quickly and exceeding disk capacity.
+Full disks can lead to issues that are hard to rectify.
+The hard limit is the maximum storage limit.
```

Comment:
How to check? Check the logs if producers start to slow?

Reply:
Right now, there is only a log message in the Kafka log as far as I know. In the future there should be also some metrics, but not there yet.